Create arrow function/script.js

```diff
@@ -0,0 +1,4 @@
+const greet = (val) => {
+  return `Hi, ${val}!`;
+  greet('cool coders');
```

Comment:
@glecie18 Sorry I missed this one, Please move this line outside the arrow function, Like so

```
const greet = (val) => {
  return `Hi, ${val}!`;
};
greet("cool coders");

```

Reason: You already returned a value on line 2, Therefore line 3 won't get execute or will be skipped.